tests/test_schema.py

```diff
@@ -90,13 +90,6 @@ def test_set_elem_propogate(self):
         res_schema.set_schema_elem(
                 ("properties", "b", "properties", "hello"),
                 {"type": "string"},
-                propogate=True)
+                propagate=True)
 
         print_and_test(expected, res_schema.schema)
-
-        # Deleting the elem again should match our original schema
-        res_schema.delete_group_from_schema(
-                ("properties", "b", "properties", "hello"),
-                propogate=True)
-
-        print_and_test(schema.schema, res_schema.schema)
```

Comment:
Did you accidentally remove this in the most recent commit?

Reply:
Looks like it got lost in the rebase. I've added it back in.